CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ Features listed in the internal section are not ready to use but, in the spirit
 
 ### Bug fixes:
 - Replicated references to newly created dynamic subobjects will now be resolved correctly.
+- Cloud deployment flow will now correctly report errors when a deployment fails to launch.
```

Comment:
Perhaps could be more specific here? This does usually happen but now does for an additional failure case.